btcpay-restore.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+
+if [ "$(id -u)" != "0" ]; then
+  echo "This script must be run as root."
+  echo "Use the command 'sudo su -' (include the trailing hypen) and try again"
+  exit 1
+fi
+
+backup_file=$1
+if [ -z "$backup_file" ]; then
+  echo "Usage: btcpay-restore.sh /path/to/backup.tar.gz"
+  exit
+fi
+
+if [ ! -f "$backup_file" ]; then
+  echo "$backup_file does not exist."
+  exit
+fi
+
+volumes_dir=/var/lib/docker/volumes
```

Comment:
Best not to hardcode

Reply:
in the PR draft it is mentioned as todo :)